skrub/tests/test_interpolation_join.py

```diff
@@ -76,117 +91,63 @@ def transform(self, X):
     assert_array_equal(ns.col(join, "B"), [0, 1])
 
 
-def test_no_multioutput(df_module, buildings, weather):
```

Comment:
why do we remove this test?

Reply:
The test wasn't checking for no multioutput, since we use HGBT by default and they are multioutput

I will add a specific test